roles/dcos_cli/vars/promdash.yml

```diff
@@ -0,0 +1,4 @@
+dcos_cli_app_promdash_enabled: "{{ promdash_enabled }}"
```

Comment:
you are right, we can use prometheus_enabled to promdash, I am going to change it